### PR TITLE
LPS-81283 avoid null pointer

### PR DESCRIPTION
**Pull request by linolaoj**
cc: @andreldm

### CODE REVIEW

File: modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/display/context/DDLViewRecordsDisplayContext.java [hunk 1]
```diff
@@ -98,7 +99,17 @@ public DDLViewRecordsDisplayContext(
 
 		_formDDMTemplateId = formDDMTemplateId;
 
-		_user = (User)_liferayPortletRequest.getAttribute(WebKeys.USER);
+		User user = (User)_liferayPortletRequest.getAttribute(WebKeys.USER);
```
Comment (rafaprax) [inline modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/display/context/DDLViewRecordsDisplayContext.java]: Hey @linolaoj Can you use the `PortalUtil.getUser` instead?

Comment (linolaoj) [reply modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/display/context/DDLViewRecordsDisplayContext.java]: pushed changes

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rafaprax** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/58921. [See changes here.](https://github.com/rafaprax/liferay-portal/compare/911e44ca007b~1...911e44ca007b) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**